test/build_defs/test.build_defs

```diff
@@ -8,7 +8,7 @@ def please_repo_e2e_test(
         expected_output: dict,
 ):
     test_cmd = " && ".join(
-        ["cd $DATA_REPO", f"$TMP_DIR/$DATA_PLEASE {plz_command}"] +
+        ["cd $DATA_REPO", "export PLZ=$TMP_DIR/$DATA_PLEASE", f"$PLZ {plz_command}"] +
```

Comment:
I'd probably just change this to `"export PLZ=$TMP_DIR/$DATA_PLEASE", f"{plz_command}"`

I think this is more flexible.